### DISCUSSION TITLE
Toolbar missing?

### DISCUSSION THREAD

**Discussion by stantobob** [helgoboss/helgobox#1350] [Q&A] [answered]
Hello! I just purchased Playtime 2 and I seem to be missing the toolbar with the metronome and such things. I can't seem to find any option to toggle it or solutions online. Does anybody know a possible solution?

**Comment by helgoboss** [comment, marked as answer]
Maybe you accidentally hid the toolbar by clicking the small grey horizontal bar on the very right of the window title bar? If not, please post a screenshot.

**Reply by stantobob** [reply]
That was it! Thank you very much!
